plugins/messages/app/mailers/messages_mailer.rb

```diff
@@ -6,6 +6,7 @@ def foodsoft_message(message, recipient)
 
     mail subject: "[#{FoodsoftConfig[:name]}] " + message.subject,
          to: recipient.email,
-         from: "#{show_user(message.sender)} <#{message.sender.email}>"
+         from: "#{show_user(message.sender)} via Foodsoft <#{FoodsoftConfig[:email_sender]}>",
```

Comment:
I'd suggest keeping it hardcoded for now, since we have an even bigger problem: The text depends on the language of the sender and not of the recipient.